README.md

```diff
@@ -205,6 +205,18 @@ Run blazepack in verbose mode and share the log by raising an issue
 ```
 blazepack start --verbose > blazepack.log
 ```
+### Contributing To This Project
+
+```
+Fork this repository to make your own copy
+git clone https://github.com/yourusername/blazepack
+cd blazepack
+npm install
+npm link 
```

Comment:
actually you can create a playground folder (which is git ignored) inside the codebase and use `node ../bin/blazepack.js create my-cra` inside the playground folder also to test things as npm link would mess with your local installation of blazepack and it is very flaky.

Reply:
Nice idea!!